src/sundials/shared/sundialsmath.c

```diff
@@ -39,12 +39,18 @@ realtype RPowerI(realtype base, int exponent)
   return(prod);
 }
 
+extern double hoc_pow(double, double);
+extern double hoc_c_pow(double, double);
+double hoc_c_pow(double x, double y) {
+  return hoc_pow(x, y);
+}
```

Comment:
It's not immediately obvious to me why this function is needed. Can `hoc_c_pow` be replaced with `hoc_pow`?

Reply:
only hoc_pow is needed.